Firmware/temperature.cpp

```diff
@@ -1459,12 +1480,11 @@ void min_temp_error(uint8_t e) {
 	return;
 #endif
 //if (current_temperature_ambient < MINTEMP_MINAMBIENT) return;
-  disable_heater();
```

Comment:
also keep the disable_heater() here

Reply:
Stop() does that of course as the first call. Maybe we should move Stop() as the first action instead?